lib/livebook/session.ex

```diff
@@ -901,6 +905,8 @@ defmodule Livebook.Session do
       session = self_from_state(state)
 
       with :ok <- Livebook.Tracker.track_session(session) do
```

Comment:
Wait, what happens if this does not match? Doesn't it mean we never receive?

Reply:
We would return `{:error, _}` from init, so the caller wouldn't wait for a message either :)

Reply:
Oh, doh!